src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -0,0 +1,34 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+    private final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/car/driver/add.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Car car = carService.get(Long.parseLong(req.getParameter("car_id")));
+        Driver driver = driverService.get(Long.parseLong(req.getParameter("driver_id")));
+        carService.addDriverToCar(driver, car);
```

Comment:
```suggestion
        carService.addDriverToCar(driver, car);
        resp.sendRedirect("/cars/drivers/add")
```

Lets display something on web page after adding driver to car. For example you can show page with adding driver to car form, so that the user can add another driver to the car.